src/identity/PublicIdentity.ts

```diff
@@ -44,7 +55,18 @@ export default class PublicIdentity implements IPublicIdentity {
       return undefined
     }
   }
-
+  /**
+   * @description (STATIC) (ASYNC) Resolves internal and external DIDs (Decentralised identifier) from a specific identifier
+   * @param identifier URL scheme identifier
+   * @param urlResolver  DID URL always identifies the resource to be located
+   * @returns  A new [[PublicIdentity]] object
+   * @example
+   * ```javascript
+   *
+   * PublicIdentity.resolveFromDid(identifier, urlResolver)
```

Comment:
There should be an assignment (see my comment on `didDocument`).

Reply:
I have updated it 👍